pyproject.toml

```
@@ -50,8 +50,11 @@ environs = "9.5.0"
 python-crontab = "3.0.0"
 # needed for liquid support boltz
 wallycore = {version = "^1.0.0", optional = true}
+# needed for breez funding source
+breez-sdk = {version = "0.2.14", optional = true}
```

Can be bumped to 0.2.15
```suggestion
breez-sdk = {version = "0.2.15", optional = true}
```